src/core/server/status/plugins_status.ts

```diff
@@ -331,11 +331,15 @@ export class PluginsStatusService {
    */
   private updatePluginReportedStatus(plugin: PluginName, reportedStatus: ServiceStatus): void {
     const previousReportedLevel = this.pluginData[plugin].reportedStatus?.level;
+    const previousReportedSummary = this.pluginData[plugin].reportedStatus?.summary;
 
     this.pluginData[plugin].reportedStatus = reportedStatus;
     this.pluginStatus[plugin] = reportedStatus;
 
-    if (reportedStatus.level !== previousReportedLevel) {
+    if (
+      reportedStatus.level !== previousReportedLevel ||
```

Comment:
should we compare the whole reportedStatus to previoudReportedStatus here?

Reply:
It might make us less prone to breakage if compare the whole status object here, yes. What does this object look like? Is it simple enough to do a shallow equality check here?